authz/sdk_end2end_test.go

```diff
@@ -379,6 +379,30 @@ func (s) TestSDKFileWatcherEnd2End(t *testing.T) {
 	}
 }
 
+func verifyValidReload(ctx context.Context, tsc pb.TestServiceClient, wantCode codes.Code, wantErr string) (lastStatus *status.Status) {
+	for numRetries := 0; numRetries <= 20; numRetries++ {
```

Comment:
Why not use `ctx` instead?  `20` is very arbitrary.  The caller said they were willing to wait for the context to expire so using context seems better to me.

Reply:
Sounds good. Modified the function.